The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/26891300415/job/79317975029) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/79317975029) [(plain)](https://github.com/rust-lang/rust/commit/1ec9b30733cf32da9bd45b6f00faeac67bdcc53f/checks/79317975029/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
............................................       (144/144)

======== tests/rustdoc-gui/huge-logo.goml ========

[ERROR] line 10: The following errors happened (for selector `.sidebar-crate .logo-container img`): [expected `16px` for key `border-top-width`, found `0px`]: for command `assert-css: (".sidebar-crate .logo-container img", {"border-top-width": "16px", "margin-top": "-16px"})`
    at <file:///checkout/obj/build/x86_64-unknown-linux-gnu/test/rustdoc-gui/doc/huge_logo/index.html>


<= doc-ui tests done: 143 succeeded, 1 failed, 0 filtered out

Error: ()
```

</details>

--- a/tests/rustdoc-gui/huge-logo.goml
+++ b/tests/rustdoc-gui/huge-logo.goml
@@ -7,4 +7,4 @@ set-window-size: (1280, 1024)
 assert-property: (".sidebar-crate .logo-container", {"offsetWidth": "96", "offsetHeight": 48})
 // offsetWidth = width of sidebar, offsetHeight = height + top padding
 assert-property: (".sidebar-crate .logo-container img", {"offsetWidth": "48", "offsetHeight": 64})
-assert-css: (".sidebar-crate .logo-container img", {"border-top-width": "16px", "margin-top": "-16px"})
+assert-css: (".sidebar-crate .logo-container img", {"padding-top": "16px", "margin-top": "-16px"})

